napari/utils/colormaps/vendored/cm.py

```diff
@@ -189,7 +189,6 @@ class ScalarMappable(object):
     """
```

Comment:
Just flagging that this change happens inside our `vendored` folder which we might want to ignore. Thoughts on this @jni? We could easily revert the few changes inside vendored files before merging, but to you

Reply:
oh, yeah, I can revert that or redo just the part not in vendor.

Reply:
done.